### PR TITLE
fix: allow smart contract to remove liquidity

### PR DESCRIPTION
**Pull request by berteotti**
# Description

Smart Contract accounts seem to not deal well with off-chain approvals. This fixes remove liquidity by doing the approval on-chain

### CODE REVIEW

File: src/pages/Pools/RemoveLiquidity/index.tsx [hunk 1]
```diff
@@ -131,14 +130,15 @@ export default function RemoveLiquidity() {
   const routerAddress = UniswapV2RoutablePlatform.SWAPR.routerAddress[chainId ? chainId : ChainId.MAINNET]
   const [approval, approveCallback] = useApproveCallback(parsedAmounts[Field.LIQUIDITY], routerAddress)
 
-  const isArgentWallet = useIsArgentWallet()
-
   async function onAttemptToApprove() {
-    if (!pairContract || !pair || !provider || !deadline) throw new Error('missing dependencies')
+    console.log('hello', 'clicked')
```
Comment (Diogomartf) [inline src/pages/Pools/RemoveLiquidity/index.tsx:134]: console.log!

```suggestion
```

File: src/pages/Pools/RemoveLiquidity/index.tsx [hunk 2]
```diff
@@ -180,11 +180,13 @@ export default function RemoveLiquidity() {
       primaryType: 'Permit',
       message,
     })
+    console.log('hello', message)
```
Comment (Diogomartf) [inline src/pages/Pools/RemoveLiquidity/index.tsx:183]: if we need the logs we need to have proper names. So we can know which message is.

For eg.
```suggestion
    console.log('onAttemptToApprove message', message)
```

I'll do the same for all of the logs.

Comment (berteotti) [reply src/pages/Pools/RemoveLiquidity/index.tsx]: Just cleaned all the logs. We don't need them :)

**Review by Diogomartf — APPROVED** [review]

File: src/pages/Pools/RemoveLiquidity/index.tsx (hunk 1 above)
Comment (berteotti) [reply src/pages/Pools/RemoveLiquidity/index.tsx]: Cleaned them all, thanks!

**Pull request merged**